src/applications/mhv-supply-reordering/components/Breadcrumbs.jsx

```diff
@@ -9,14 +8,23 @@ const breadcrumbList = [
   },
   {
     href: '/my-health',
-    label: 'Health care',
+    label: 'My HealtheVet',
   },
   {
     href: manifest.rootUrl,
     label: manifest.appName,
   },
 ];
+const bcString = JSON.stringify(breadcrumbList);
```

Comment:
It's only necessary to perform this operation when using the non-react-based web component. You can pass `VaBreadcrumbs`, the react wrapper for VA web components the JS array.

Reply:
https://design.va.gov/storybook/?path=/docs/uswds-va-breadcrumbs--docs

Reply:
And, now I realize you're probably doing it this way because you would rather have a test-id on the web-component, instead of having to add another `<div/>`